server/utils.go

```diff
@@ -20,7 +20,7 @@ func (p *Plugin) CreateBotDMPost(userID string, post *model.Post) (*model.Post,
 	if post.Props == nil {
 		post.Props = make(map[string]interface{})
 	}
-	post.Props["from_webhook"] = true
+	post.Props["from_webhook"] = "true"
```

Comment:
I'm not convinced that you want this. Messages from Bot Accounts will have a label anyway. Adding this prop _might_ cause problems.

Reply:
Actually, the messages from the bot account don't have the label from what I've seen. When I was demoing the feature yesterday, the BOT label was not being applied to some of the posts because this was missing using the incorrect type.

Reply:
Sorry, I was ambiguous. The code to add the label isn't implemented jet.

But let's see how this will work out, once bot accounts are fully implemented.